opal/mca/common/ucx/common_ucx_wpool.c

```diff
@@ -282,6 +283,10 @@ opal_common_ucx_wpool_progress(opal_common_ucx_wpool_t *wpool)
         } while (progressed);
```

Comment:
We can skip the default worker here to avoid double-progress.

But we can't skip progressing the default worker if there are active workers. The default worker might still not be active, but you want to progress it.